src/Notify/index.js

```diff
@@ -7,7 +7,10 @@ import NotifyMesssage from './NotifyMesssage'
 export default class Notify extends Component {
   static propTypes = {
     notifications: PropTypes.arrayOf(PropTypes.object).isRequired,
+    handleMessageClick: PropTypes.func,
```

Comment:
onMessageClick and isSticky should not be passed down from the Notify container to indiviual notifications. The notifications array should instead contain this data